hoot-core/src/main/cpp/hoot/core/conflate/poi-polygon/extractors/PoiPolygonAddressScoreExtractor.h

```diff
@@ -72,58 +72,22 @@ class PoiPolygonAddressScoreExtractor : public FeatureExtractorBase, public Conf
   virtual double extract(const OsmMap& map, const ConstElementPtr& poi,
                          const ConstElementPtr& poly) const;
 
-  /**
-   * Determines if a node has an address
-   *
-   * @param node the node to examine for an address
-   * @return true if the node has an address; false otherwise
-   */
-  static bool nodeHasAddress(const Node& node);
-
-  /**
-   * Determines if an element has an address
-   *
-   * @param element the element to examine for an address
-   * @param map map the element being examined belongs to
-   * @return true if the element has an address; false otherwise
-   */
-  static bool elementHasAddress(const ConstElementPtr& element, const OsmMap& map);
-
   virtual QString getDescription() const
   { return "Scores address similarity for POI/Polygon conflation"; }
 
   long getAddressesProcessed() const { return _addressesProcessed; }
   bool getMatchAttemptMade() const { return _matchAttemptMade; }
-
-  static QString getAddressTagValue(const Tags& tags, const QString addressTagType);
+  void setAllowLenientHouseNumberMatching(bool allow)
+  { _addressParser.setAllowLenientHouseNumberMatching(allow); }
 
```

Comment:
"lenient house number matching" was formerly referred to as "subletter matching"